go/arrow/decimal256/decimal256_test.go

```diff
@@ -575,3 +577,31 @@ func TestFromString(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Could you add some tests which hit the bitlen overflow?

Reply:
added!